index.js

```diff
@@ -50,7 +61,18 @@
 // // => "different"
 // console.log("%c----------", "color: red") 
 
-
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum) {
+    return "same same"
+  }
+  else if (num == maybeNum) {
+    return "same same (but different)"
+  }
+  else {
+    return "different"
+  }
+}
```

Comment:
Nice! Another solution would be to convert the string to a number with parseInt to avoid using `==` in the second conditional statement :

`} else if (num === parseInt(maybeNum)) {`